tests/unit/test_store.py

```diff
@@ -15,15 +16,15 @@ def test_npy_persisted_array():
     ones = np.ones((10,2))
     append2 = np.random.rand(23, 2)
 
-    arr = NpyPersistedArray(filename, truncate=True)
+    arr = NpyArray(filename, truncate=True)
     arr.append(original)
     assert np.array_equal(original, arr[:])
     arr.close()
```

Comment:
Sure, but at the moment they are not implemented. If you have time, that would be a nice addition.